README.md

```diff
@@ -41,7 +41,8 @@ python manage.py createsuperuser
 python manage.py runserver
 ```
 
-For detailed installation and setup docs, see [docs.wagtail.org](https://docs.wagtail.org/).
+Note: Above commands are the implementation of Wagtail, NOT setup guide.
```

Comment:
I'm not sure that this really makes things clearer... I don't know what "the implementation of Wagtail" is meant to mean (to me that would mean the Python code of Wagtail itself), and a beginner would be even more confused about why this is here, if it's something else besides setup instructions.

Maybe this is just missing an instruction to go to `http://localhost:8000`, which will then bring up the welcome page with further documentation links. Or maybe we don't really need this section at all, and should just link to the Getting Started page of the docs. I understand there's some marketing appeal in saying "Wagtail can be installed in just 7 lines", but that's already on the website, and the Getting Started docs do a better job of guiding users through setup, including setting up a virtual environment.

cc @KalobTaulien who's working on some wider improvements to the readme.